app/controllers/CyaCategorisationController.scala

```diff
@@ -36,15 +41,62 @@ class CyaCategorisationController @Inject() (
     with I18nSupport
     with Logging {
 
-  def onPageLoad(): Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
-    val list = SummaryListViewModel(
-      rows = Seq.empty
-    )
-    Ok(view(list))
+  def onPageLoad(recordId: String): Action[AnyContent] = (identify andThen getData andThen requireData) {
+    implicit request =>
+      CategorisationAnswers.build(request.userAnswers, recordId) match {
+        case Right(_) =>
+          val categorisationAnswers = for {
+            recordCategorisations <- request.userAnswers.get(RecordCategorisationsQuery)
+            categorisationAnswers <- recordCategorisations.records.get(recordId)
+          } yield categorisationAnswers
+
+          val categorisationRows = categorisationAnswers match {
+            case Some(categorisationInfo) =>
+              categorisationInfo.categoryAssessments
+                .flatMap(assessment =>
+                  AssessmentsSummary.row(
+                    recordId,
+                    request.userAnswers,
+                    assessment,
+                    categorisationInfo.categoryAssessments.indexOf(assessment),
+                    categorisationInfo.categoryAssessments.size
+                  )
+                )
+            case None                     => Seq.empty
```

Comment:
is this case possible/valid?

Reply:
I was thinking if there were no assessments on the categorisation but I guess we don't end up at check your answers in that case? 
Unless you can enter a longer commodity / supplementary unit even if there are no assessments? 

Dunno basically